src/main/java/org/llorllale/cactoos/matchers/ScalarHasValue.java

```diff
@@ -56,32 +48,14 @@ public ScalarHasValue(final T value) {
 
     /**
      * Ctor.
-     * @param mtr Matcher of the text
+     * @param mtr Matcher of the value
      */
     public ScalarHasValue(final Matcher<T> mtr) {
-        super();
-        this.matcher = mtr;
-    }
-
-    @Override
-    public void describeTo(final Description description) {
-        description.appendText("Scalar with ");
-        description.appendDescriptionOf(this.matcher);
-    }
-
-    // @checkstyle ProtectedMethodInFinalClassCheck (1 line)
-    @Override
-    protected boolean matchesSafely(final Scalar<T> item) {
-        return this.matcher.matches(
-            new Unchecked<>(item).value()
+        super(
+            // @checkstyle IndentationCheck (3 line)
+            slr -> mtr.matches(slr.value()),
```

Comment:
@victornoel `slr` is very obscure. How about just `scalar` (it's Scalar, right? :))